accounts-db/accounts-hash-cache-tool/src/main.rs

```diff
@@ -825,6 +797,27 @@ fn open_file(
     Ok((reader, header))
 }
 
+/// Prints unique entries
+fn print_unique_entries(entries: &[CacheHashDataFileEntry], lamports_width: usize) {
+    if entries.is_empty() {
+        println!("(none)");
+        return;
+    }
+
+    let count_width = width10(entries.len() as u64);
+    let mut total_lamports = Saturating(0);
+    for (i, entry) in entries.iter().enumerate() {
+        total_lamports += entry.lamports;
+        println!(
```

Comment:
nit: probably, it is worth to  implement a customized Display trait on `CacheHashDataFileEntry`. 
Then, we can simplify print by printing entry directly.